Add a notice to temporarily refrain from translating (#105)
python/python-docs-pl: README.en.md

![3 Translators](https://img.shields.io/badge/Translators-3-0.svg)
<!-- [[[end]]] -->

[*Przeczytaj to w języku polskim*](README.md)

> [!IMPORTANT]  

> > Please temporarily refrain from translating on the Transifex platform, the changes made may be lost.

Reviewer: ```suggestion
> Please temporarily refrain from translating on the Transifex platform, the translations will be lost.
```
